app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/EditEvent.kt

```diff
@@ -0,0 +1,311 @@
+                          // Upload image and update event
+                          val finalEvent =
+                              if (imageUri != null) {
+                                val imageUrl = eventViewModel.uploadImageAndGetUrl(imageUri!!)
+                                val updatedImages = event!!.images.toMutableList()
+                                if (updatedImages.isNotEmpty()) {
+                                  updatedImages[0] = imageUrl
+                                } else {
+                                  updatedImages.add(imageUrl)
+                                }
+                                updatedEvent.copy(images = updatedImages)
+                              } else {
+                                updatedEvent
+                              }
+                          eventViewModel.editEvent(finalEvent)
+                          refreshEvent(finalEvent)
+                        }
+                      }
+                    })
+          }
+        },
+        bottomBar = {
+          BottomNavigationMenu(
+              onTabSelect = { selectedTab ->
+                nav.navigateTo(TOP_LEVEL_DESTINATIONS.first { it.route == selectedTab })
+              },
+              tabList = TOP_LEVEL_DESTINATIONS,
+              selectedItem = Route.EVENTS)
+        },
+        content = { innerPadding ->
+          Box(modifier = Modifier.padding(innerPadding)) {
+            Column(
+                modifier =
+                    Modifier.padding(start = 15.dp, end = 15.dp)
+                        .verticalScroll(rememberScrollState(0))
+                        .fillMaxSize(),
+                verticalArrangement = Arrangement.Top,
+                horizontalAlignment = Alignment.CenterHorizontally) {
+                  Image(
+                      painter =
+                          if (imageBitmap != null) {
+                            androidx.compose.ui.graphics.painter.BitmapPainter(imageBitmap!!)
+                          } else if (!profilePictureUrl.isNullOrEmpty()) {
+                            rememberAsyncImagePainter(profilePictureUrl)
+                          } else {
+                            painterResource(id = R.drawable.gomeet_logo)
+                          },
+                      contentDescription = "Event picture",
+                      modifier =
+                          Modifier.padding(start = 15.dp, end = 15.dp, top = 30.dp, bottom = 15.dp)
+                              .width(101.dp)
+                              .height(101.dp)
+                              .clickable { imagePickerLauncher.launch("image/*") }
+                              .clip(CircleShape)
+                              .background(color = MaterialTheme.colorScheme.background)
+                              .align(Alignment.CenterHorizontally)
+                              .testTag("Event Picture"),
+                      contentScale = ContentScale.Crop)
+
```

Comment:
This image currently looks like a profile picture, which feels a bit odd to me. What do you think about making it larger, similar to the image in EventInfo?

(Edit Event) :
<img width="294" alt="Capture d’écran 2024-05-21 à 17 32 40" src="https://github.com/SwEnt-Project-G18/SwEntApp/assets/92448776/16c5feae-0779-4b3e-a322-df5e0ea27102">

EventInfo:
<img width="319" alt="Capture d’écran 2024-05-21 à 17 33 28" src="https://github.com/SwEnt-Project-G18/SwEntApp/assets/92448776/aa949cf3-b397-4df3-9c16-83e33597244b">

Reply:
Correct, that has been changed too